src/store/index.js

```diff
@@ -5,9 +5,21 @@ import {
   SET_USER,
   SET_MESSAGES,
   SET_PAGINATION,
+  REMOVE_MESSAGE,
+  INCREMENT_PAGINATION_TOTAL,
+  DECREMENT_PAGINATION_TOTAL,
+  UPDATE_MESSAGE,
 } from './mutation-types';
 import AuthService from '@/api/services/AuthService';
 import MessagesService from '@/api/services/MessagesService';
+import formatDate from '../utils/formatDate';
+
+const formatMessages = (response) => {
```

Comment:
This name would make more sense since this function is not aware of any api related stuff
```suggestion
const formatMessages = (messages) => {
```